python/ray/serve/scripts.py

```diff
@@ -397,11 +403,40 @@ def build(import_path: str, app_dir: str, output_path: Optional[str]):
             if property in deployment:
                 del deployment[property]
 
-    if output_path is not None:
-        if not output_path.endswith(".yaml"):
-            raise ValueError("FILE_PATH must end with '.yaml'.")
+    config_str = yaml.dump(
+        config, Dumper=ServeBuildDumper, default_flow_style=False, sort_keys=False
+    )
+    config_str += f"# Built using Ray v{ray.__version__}\n"
```

Comment:
We should put this at the top of the file. And add a line saying "This file is generated using `serve build` command" or something like that.

Reply:
I moved the comment to the top of the file and changed it to say:

```yaml
# This file was generated using the `serve build` command on Ray v3.0.0.dev0.
```